packages/gatsby/src/utils/worker/pool.ts

```diff
@@ -1,10 +1,25 @@
 import Worker from "jest-worker"
 import { cpuCoreCount } from "gatsby-core-utils"
+import { getFilePath } from "./page-data"
+interface IGatsbyWorkerPool {
+  getFilePath: typeof getFilePath
+  renderHTML({
+    htmlComponentRendererPath,
+    paths,
+    envVars,
+  }: {
+    htmlComponentRendererPath: string
+    paths: string[]
+    envVars: Array<[string, string | undefined]>
+  }): Promise<void[]>
+}
```

Comment:
this is goofy. I wish Worker took an argument of methods to add to itself..

Could we add a comment on why this is necessary? Basically just becuase Worker self attaches methods from the ./child instance.